code/style.css

```diff
@@ -1,4 +1,149 @@
-html {
-  background: #0025ff;
-  color: #fff;
+@import url('https://fonts.googleapis.com/css2?family=Architects+Daughter&family=Bebas+Neue&family=Shadows+Into+Light&display=swap');
+body {
```

Comment:
Between all of the different styling it would be nice to see some blank lines for it to be easier to read and also less tab space :) I really recommend the Gudielines in how to write good code, it helpt me allot :)